concent_api/conductor/urls.py

```diff
@@ -3,5 +3,5 @@
 from .views import report_upload
 
 urlpatterns = [
-    url(r'^report-upload/(?P<file_path>.+\.zip)$', report_upload, name='report-upload'),
+    url(r'^report-upload/(?P<file_path>.*)$', report_upload, name='report-upload'),
```

Comment:
Please use `.+` instead of `.*` so that empty names are not allowed.

Unless there's something in the view that will reject them.

Reply:
Hm... I don't think that view would reject it. Changing.